etna/datasets/internal_datasets.py

```diff
@@ -80,25 +94,43 @@ def load_dataset(name: str, download_path: Path = _DOWNLOAD_PATH, rebuild_datase
     ------
     NotImplementedError:
         if name not from available list of dataset names
+    NotImplementedError:
+        if part not from available list of dataset parts
     """
     if name not in datasets_dict:
         raise NotImplementedError(f"Dataset {name} is not available. You can use one from: {sorted(datasets_dict)}.")
 
+    parts_ = (parts,) if isinstance(parts, str) else parts
+    dataset_params = datasets_dict[name]
+    for part in parts_:
+        if part not in dataset_params["parts"]:
+            raise NotImplementedError(f"Part {part} is not available. You can use one from: {dataset_params['parts']}.")
+
     dataset_dir = download_path / name
-    dataset_path = dataset_dir / f"{name}.csv"
+    dataset_path = dataset_dir / f"{name}_full.csv.gz"
+
+    get_dataset_function = dataset_params["get_dataset_function"]
+    freq = dataset_params["freq"]
 
-    get_dataset_function, freq = datasets_dict[name].values()
     if not _check_dataset_local(dataset_path) or rebuild_dataset:
-        ts = get_dataset_function(dataset_dir)
+        get_dataset_function(dataset_dir)
+
+    if len(parts_) == 1:
+        data = pd.read_csv(dataset_dir / f"{name}_{parts_[0]}.csv.gz", compression="gzip", header=[0, 1], index_col=[0])
```

Comment:
Why `header=[0, 1]`?

Reply:
We save data with `MultiIndex`, so we to have to define `header=[0, 1]` in `pd.read_csv` to load data correctly (first two rows of saved file are indices).